x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/signals_migration/read_signals_migration_status/read_signals_migration_status.schema.yaml

```diff
@@ -24,6 +24,7 @@ paths:
               Time from which data is analyzed. For example, now-4200s means the rule analyzes data from 70 minutes
               before its start time. Defaults to now-6m (analyzes data from 6 minutes before the start time).
```

Comment:
I've noticed this API has wrong method: https://github.com/elastic/kibana/blob/f5b71958d8be26563174381bf381256602e76f58/x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/signals_migration/read_signals_migration_status/read_signals_migration_status.schema.yaml#L7

It should be GET, not POST. Reference - https://www.elastic.co/guide/en/security/current/signals-migration-api.html#migration-1

Reply:
Great catch! Updated.